lte/gateway/docker/README.md

```diff
@@ -40,7 +40,7 @@ containerized AGW by running the following steps inside the VM:
 ```
 cd $MAGMA_ROOT/lte/gateway && make run  # You can skip this if you have built the AGW with make before
 for component in redis nghttpx td-agent-bit; do cp "${MAGMA_ROOT}"/{orc8r,lte}/gateway/configs/templates/${component}.conf.template; done
-sudo systemctl stop 'magma@*' 'sctpd' # We don't want the systemd-based AGW to run when we start the containerized AGW
+sudo systemctl stop 'magma@*' 'sctpd' 'redis' # We don't want the systemd-based AGW to run when we start the containerized AGW
```

Comment:
I think this is not needed as the Redis service used by magma is called `magma@redis.service`.

Reply:
You're right, sorry for the noise!